vendor/modules.txt

```diff
@@ -191,6 +191,9 @@ github.com/prometheus/common/model
 github.com/prometheus/procfs
 github.com/prometheus/procfs/internal/fs
 # github.com/rook/rook v1.2.4
+github.com/rook/rook/pkg/apis/cassandra.rook.io
+github.com/rook/rook/pkg/apis/cassandra.rook.io/v1alpha1
+github.com/rook/rook/pkg/apis/ceph.rook.io
```

Comment:
Doesn't seem related to this PR.

Reply:
this was updated after `make deps-update` (`go mod vendor`) probably something that was updated in a different change.
I don't think I have control over it